app/grants/models.py

```diff
@@ -590,6 +608,33 @@ def successful_contribution(self, tx_id):
         except ConversionRateNotFoundError as e:
             logger.info(e)
 
+        if self.num_tx_processed == self.num_tx_approved:
+            try:
+                converted_amount = (
+                    int(convert_amount(
+                        self.amount_per_period,
+                        self.token_symbol,
+                        "USDT")
+                    )
+                )
+
+                if self.frequency_unit == 'days':
```

Comment:
it'd be nice if `period_seconds` was its own method too.

also the `ConversionRateNotFoundError` try /catch should probably not wrap this